panels/AssetPanel.php

```diff
@@ -50,11 +50,7 @@ public function getDetail()
      */
     public function save()
     {
-        try {
-            $bundles = Yii::$app->view->assetManager->bundles;
-        } catch (\Exception $e) {
-            $bundles = null;
-        }
+        $bundles = Yii::$app->view->assetManager->bundles;
```

Comment:
This changes behavior; catching the exception outside the panel will cause it to show up in the exceptions panel.